AndroidSDK/src/com/leanplum/LeanplumUIEditor.java

```diff
@@ -26,6 +26,7 @@
 /**
  * Describes the API of the visual editor package.
  */
+@Deprecated
```

Comment:
Please add a comment to the javadoc like: `Deprecation: This method will become package protected in a future release, since t is not intended to be public API.`